INSTALL.md

```diff
@@ -1,149 +1,123 @@
-Installation overview
```

Comment:
I agree, but my focus was more on writing it and having it merged first, then we can move it.. would that work?